Web Development/README.md

```diff
@@ -348,6 +348,11 @@ to everyone - everywhere!</td>
         <td>Hitesh Choudhari is a YouTube channel that offers various courses on programming languages. It is one of the best channel to learn web development at free of cost.
         </td>
       </tr>
+      <tr>
+        <td><a href="">https://www.youtube.com/watch?v=bMknfKXIFA8</a>FreeCodeCamp</td>
```

Comment:
Another problem is this. Please change this code to this one:
`<td><a href="https://www.youtube.com/watch?v=bMknfKXIFA8">React Course - Beginner's Tutorial for React JavaScript Library [2022]</a></td>`

Reply:
Done